### ISSUE TITLE
Offer MUSL builds

### ISSUE THREAD

**Issue by ofek** [nickgerace/gfold#260]
This would allow downloads for systems with an older glibc than what is currently used to build the Linux binaries.

**Comment by nickgerace** [comment]
Good idea! `gfold` used to do this a long time ago, though I can look into the current state of cross-compilation and GitHub action runner OS offerings.

**Comment by ofek** [comment]
Has this been implemented?

**Comment by nickgerace** [comment]
Now it has!

**Comment by nickgerace** [comment]
You're welcome! Please let me know if you run into an issue with them
